linode/lke/framework_datasource.go

```diff
@@ -69,6 +69,20 @@ func (r *DataSource) Read(
 		return
 	}
 
+	var dashboard *linodego.LKEClusterDashboard
+
+	// LKE Enterprise does not have a dashboard URL.
+	if cluster.Tier != TierEnterprise {
```

Comment:
optional: Would it make more sense to check if the tier is explicitly `standard` here to proof us for potential future tiers?